File: 3737-count-subarrays-with-majority-element-i/3737-count-subarrays-with-majority-element-i.py
class Solution:
    def countMajoritySubarrays(self, nums: List[int], target: int) -> int:
        cnt_subarray=0
        n=len(nums)
        for i in range(n):
            cnt_target=0
            for j in range(i,n):
                if (nums[j]==target):
                    cnt_target+=1
                if (j-i+1< 2*cnt_target) :
                    cnt_subarray+=1
        return cnt_subarray

File: 3737-count-subarrays-with-majority-element-i/3737-count-subarrays-with-majority-element-i.cpp
class Solution {
public:
    int countMajoritySubarrays(vector<int>& nums, int target) {
        int cnt_subarray=0;
        int n=nums.size();
        for(int i=0; i<n;i++){
            int target_cnt=0;
            for(int j=i;j<n;j++){
                if(nums[j]==target) target_cnt++;
                if(j-i+1<2*target_cnt) cnt_subarray++;
            }
        }
        return cnt_subarray;
    }
};